src/AltinnCore/Runtime/Controllers/InstanceController.cs

```diff
@@ -332,6 +350,19 @@ public async Task<IActionResult> StartService(StartServiceModel startServiceMode
                 // Create a new instance document                
                 Instance instance = await _instance.InstantiateInstance(startServiceModel, serviceModel, serviceImplementation);
 
+                // Create and store the instance created event
+                InstanceEvent instanceEvent = new InstanceEvent
+                {
+                    AuthenticationLevel = requestContext.UserContext.AuthenticationLevel,
+                    EventType = InstanceEventType.Created.ToString(),
+                    InstanceId = instance.Id,
+                    InstanceOwnerId = instanceOwnerId.ToString(),
```

Comment:
Is there a reason for using InstanceOwnerId on the InstanceEvent. Is not that given?

Reply:
Other than it being able to retrieve solely based on a query in this collection, no. But keeping it for now as we don't know which queries we will require in the future.